Config.cmake.in

```diff
@@ -16,6 +16,12 @@ find_dependency(Python3 COMPONENTS Interpreter Development)
 find_dependency(Boost COMPONENTS log)
 find_dependency(HighFive)
 
+# ROOT is an optional dependency so we use find_package
+find_package(ROOT 6.20 QUIET)
+if (ROOT_FOUND)
+  set(fire_USE_ROOT YES)
+endif()
+
```

Comment:
Invert this order. Import the `fire_USE_ROOT` from the package and then use `find_dependency` if `fire_USE_ROOT` is true.